app.go

```diff
@@ -644,12 +645,34 @@ func (h *Headscale) getTLSSettings() (*tls.Config, error) {
 		if !strings.HasPrefix(h.cfg.ServerURL, "https://") {
 			log.Warn().Msg("Listening with TLS but ServerURL does not start with https://")
 		}
+
+		var clientAuthMode tls.ClientAuthType
+		if h.cfg.TLSClientAuthMode == "disabled" {
```

Comment:
This should be a `switch` statement, and the alternatives should be Constants.

Reply:
Good call. I'm new to Go, so thank you for this critique. Much less typing in a switch statement.

Reply:
In the latest commit, I moved this logic to a new function to make testing a bit simpler and implemented a switch statement.